src/components/base/outline-modal/outline-modal.css

```diff
@@ -0,0 +1,40 @@
+* {
+  @apply box-border;
```

Comment:
Not 100% sure this is needed, but I could be wrong.

Reply:
Thanks for taking a look!

I don't know either, but I saw it in a best-practices. Maybe it's part of the global styles? I can try removing.

Reply:
It didn't seem to do anything here, so I removed it. Thanks again.

Reply:
Yeah I think that is part of global styles.